src/provider/extensions/starknetId.ts

```diff
@@ -58,6 +85,15 @@ export class StarknetId {
     );
   }
 
+  /**
+   * Retrieves the Starkname associated with a given Ethereum address.
+   *
+   * @param {ProviderInterface} provider - The Ethereum provider interface.
```

Comment:
```suggestion
   * @param {ProviderInterface} provider - The provider interface.
```

please check all instances of "Ethereum" and remove!